src/test/java/mil/dds/anet/test/resources/RollupGraphComparatorTest.java

```diff
@@ -0,0 +1,71 @@
+package mil.dds.anet.test.resources;
```

Comment:
This class appears not to be a test of a resource class, so should be in a different package.

Reply:
Sort of agree and disagree. While not strictly a resource class, it is an inner class of a resource class.